src/server/ApiMessage.h

```diff
@@ -73,4 +81,6 @@ typedef struct _CONSOLE_API_MSG
     void SetReplyStatus(const NTSTATUS Status);
     void SetReplyInformation(const ULONG_PTR pInformation);
 
+    // DO NOT PUT MORE FIELDS DOWN HERE. They will be stomped by driver fills.

```

Comment:
"stomped by driver fills" is as readable as "cant use, below user32"

Reply:
propose "The tail end of this structure has a console driver packet copied over it."

Reply:
good call, thx